system-addon/data/content/tippytop/top_sites.json

```diff
@@ -11,33 +11,13 @@
   },
   {
     "title": "amazon",
-    "url": "https://www.amazon.ca/",
-    "image_url": "amazon-ca@2x.png"
-  },
-  {
-    "title": "amazon",
-    "url": "https://www.amazon.co.uk/",
-    "image_url": "amazon-uk@2x.png"
-  },
-  {
-    "title": "amazon",
-    "url": "https://www.amazon.com/",
-    "image_url": "amazon-com@2x.png"
-  },
-  {
-    "title": "amazon",
-    "url": "https://www.amazon.de/",
-    "image_url": "amazon-de@2x.png"
-  },
-  {
-    "title": "amazon",
-    "url": "https://www.amazon.fr/",
-    "image_url": "amazon-fr@2x.png"
+    "urls": ["https://www.amazon.ca/", "https://www.amazon.co.uk/", "https://www.amazon.com/", "https://www.amazon.de/", "https://www.amazon.fr/"],
```

Comment:
Sorry, I haven't been watching this repo as much recently, but why are we duplicating some tippy-top functionality here? It's already out of sync w/ tippy top here, re: smile.amazon.com.

Wouldn't it be easier to just add another flag to tippy-top JSON for `"default_site": true` or something and then do a quick `.filter()` if we just want the subset?

Reply:
yeah, we sort of branched off from tippy top master to include only default sites. But then we changed to screenshots and just brought everything over here for that because it was easier. I think we can go back to tippy top after things stabilize. Although we will never ship all the icons so I'm not sure if it even makes sense because the new icons are way different than the old composite ones.